source/donate.html.erb

```diff
@@ -0,0 +1,26 @@
+---
+title: Donate - Railsbridge Boston
+---
+
+<main>
+  <h1>Donate</h1>
+
+  <h2 class="section-title">
+    RailsBridge is run entirely on donations of money, time, and expertise.
+  </h2>
+
+  <p>
+    If your company or organization can help us make tech more welcoming and more diverse, we'd love your help. We need workshops sponsors (mostly to cover the cost of food), and also program sponsors so that we can help workshops happen in new places that don't yet have companies to help sponsor and host!
```

Comment:
The Donate button is actually only for individual donors, as opposed to sponsors (basically because of fees). Can we have separate CTA's on the same page?

"If your company or organization can help us make tech more welcoming and more diverse, we'd love your help. We always looking for sponsors to help cover the cost of food and drinks for the weekend."

"View Prospectus" button - http://www.railsbridgeboston.org/sponsorship_prospectus.pdf
"Email Us" button - rbbsponsorships@gmail.com

"If you'd like to contribute as an individual, you can do so via PayPal."

Change "Donate via PayPal" to "Support RailsBridge Boston". Or something like that.